src/app/search/search-results/search-results.component.ts

```diff
@@ -52,6 +54,7 @@ export class SearchResultsComponent implements OnInit {
   ngOnInit() {
     this.createTagCloud('tool');
     this.createTagCloud('workflow');
+    this.activeToolTab$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(toolsActive => (this.selectedIndex = toolsActive ? 0 : 1));
```

Comment:
Avoid "magic" numbers -- create const's like TOOLS_TAB_INDEX and WORKFLOWS_TAB_INDEX and use them here and on line 37 instead of 0 and 1.

Reply:
Done.